docs/appendix/zowe-yaml-configuration.md

```diff
@@ -0,0 +1,601 @@
+- **`storage.redis.ssl.truststore`**  
+ Specifies the truststore file used to keep other parties public keys and certificates. This is equivalent to the `CACHING_STORAGE_REDIS_SSL_TRUSTSTORE` variable in `instance.env`.
+- **`storage.redis.ssl.truststorePassword`**  
+ Specifies the password used to unlock the truststore. This is equivalent to the `CACHING_STORAGE_REDIS_SSL_TRUSTSTOREPASSWORD` variable in `instance.env`.
+- **`environment.preferIpAddress`**  
+ Set this parameter to `true`  to advertise a service IP address instead of its hostname. **Note:** this configuration is deprecated. Zowe start script will ignore this value and always set it to `false`. This is equivalent to the `APIML_PREFER_IP_ADDRESS` variable in `instance.env`.
+- **`apiml.security.ssl.verifySslCertificatesOfServices`**  
+ Specifies whether APIML should verify certificates of services in strict mode. Set to `true` will enable `strict` mode that APIML will validate both if the certificate is trusted in turststore, and also if the certificate Common Name or Subject Alternate Name (SAN) match the service hostname. This is equivalent to the `VERIFY_CERTIFICATES` variable defined in `<keystore-dir>/zowe-certificates.env`.
+- **`apiml.security.ssl.nonStrictVerifySslCertificatesOfServices`**  
+ Defines whether APIML should verify certificates of services in non-strict mode. Setting to `true` will enable `non-strict` mode where APIML will validate if the certificate is trusted in turststore, but ignore the certificate Common Name or Subject Alternate Name (SAN) check. Zowe will ignore this configuration if strict mode is enabled with `apiml.security.ssl.verifySslCertificatesOfServices`. This is equivalent to the `NONSTRICT_VERIFY_CERTIFICATES` variable defined in `<keystore-dir>/zowe-certificates.env`.
+
+#### Configure component app-server
+
+These configurations can be used under the `components.app-server` section:
+
+- **`port`**  
+ Defines the port which App Server should be started on. This is equivalent to the `ZOWE_ZLUX_SERVER_HTTPS_PORT` variable in `instance.env`. This may be defined as a valid port number or as an offset from the Gateway component's port. To define an offset enter `"+{offset}"` or `"-{offset}"` as a string. The offset must start with `+` or `-`.
+
+#### Configure component zss
+
+These configurations can be used under the `components.zss` section:
+
+- **`port`**  
+ Defines the port which ZSS should be started on. This is equivalent to the `ZOWE_ZSS_SERVER_PORT` variable in `instance.env`. This may be defined as a valid port number or as an offset from the Gateway component's port. To define an offset enter `"+{offset}"` or `"-{offset}"` as a string. The offset must start with `+` or `-`.
+- **`crossMemoryServerName`**  
+ Defines the procedure name of cross memory server.
+- **`tls`**  
+ Defines whether ZSS service has enabled TLS.
+
```

Comment:
@1000TurquoisePogs I'm not 100% sure how ZSS/ZLUX are configured in v2. Please double check this section. Thank you!